homeassistant/core.py

```diff
@@ -116,10 +116,20 @@
 
 
 def split_entity_id(entity_id: str) -> List[str]:
-    """Split a state entity_id into domain, object_id."""
+    """Split a state entity ID into domain and object ID."""
     return entity_id.split(".", 1)
 
 
+def humanize_entity_id(entity_id: str) -> str:
+    """Convert an entity ID into a human friendly version."""
+    return humanize_entity_name(split_entity_id(entity_id)[1])
+
+
+def humanize_entity_name(entity_name: str) -> str:
```

Comment:
I mean… the name is already "human"

Reply:
It's somewhere in the middle:
Entity ID: persistent_notification.invalid_config
Entity Name: invalid_config
Human friendly version: Invalid config